tools/ci/travis-install-dagmc.sh

```diff
@@ -23,10 +23,11 @@ cmake ../moab -DENABLE_HDF5=ON -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$MO
 make -j && make -j install
 cmake ../moab -DBUILD_SHARED_LIBS=OFF
 make -j install
-rm -rf $HOME/MOAB/moab
+rm -rf $HOME/MOAB/moab $HOME/MOAB/build
```

Comment:
We might as well delete the DAGMC build dir too if you're up for it. I probably should've done that in the first place.

Reply:
:+1: just added that